lua/nvim-tree/utils.lua

```diff
@@ -1,3 +1,5 @@
+local notify = vim.F.npcall(require, 'notify')
```

Comment:
what is this F thing :smile: ? it's not documented on my end, is it an internal lua api ?

Reply:
I did not see documentation, maybe because it's very simple, and yes, it's from the neovim lua internal api https://github.com/neovim/neovim/blob/master/runtime/lua/vim/F.lua

Reply:
I like the api, but if it's not documented i'd rather not use it in the plugin because it might be unstable :)

Reply:
I changed it to use `pcall`